### PR TITLE
License improvements

### PR DESCRIPTION
**Pull request by kacf**
Solves both MEN-3604 and MEN-5517.

### CODE REVIEW

File: meta-mender-core/recipes-mender/mender-artifact/mender-artifact_3.8.0.bb [hunk 1]
```diff
@@ -21,11 +21,86 @@ SRCREV = "d17209f320f0837cf591f4dea46537ac5cd8638e"
...
+    file://src/github.com/mendersoftware/mender-artifact/vendor/github.com/hashicorp/go-plugin/LICENSE;md5=d44fdeb607e2d2614db9464dbedd4094 \
+    file://src/github.com/mendersoftware/mender-artifact/vendor/github.com/hashicorp/go-sockaddr/LICENSE;md5=9741c346eef56131163e13b9db1241b3 \
+    file://src/github.com/mendersoftware/mender-artifact/vendor/github.com/hashicorp/vault/sdk/LICENSE;md5=65d26fcc2f35ea6a181ac777e42db1ea \
+    file://src/github.com/mendersoftware/mender-artifact/vendor/github.com/hashicorp/vault/api/LICENSE;md5=65d26fcc2f35ea6a181ac777e42db1ea \
+    file://src/github.com/mendersoftware/mender-artifact/vendor/github.com/hashicorp/yamux/LICENSE;md5=2dd1a9ecf92cd5617f128808f9b85b44 \
+    file://src/github.com/mendersoftware/mender-artifact/vendor/github.com/hashicorp/go-immutable-radix/LICENSE;md5=65d26fcc2f35ea6a181ac777e42db1ea \
+    file://src/github.com/mendersoftware/mender-artifact/vendor/github.com/hashicorp/go-multierror/LICENSE;md5=d44fdeb607e2d2614db9464dbedd4094 \
+    file://src/github.com/mendersoftware/mender-artifact/vendor/github.com/hashicorp/golang-lru/LICENSE;md5=f27a50d2e878867827842f2c60e30bfc \
```
Comment (kacf) [inline meta-mender-core/recipes-mender/mender-artifact/mender-artifact_3.8.0.bb:103]: @lluiscampos: This is what you were wondering before: These are generated [this script](https://github.com/mendersoftware/mendertesting/pull/209#discussion_r952238861).

**Review by lluiscampos — APPROVED** [review]
I tried but I couldn't spot anything wrong. Great work overall.

File: meta-mender-core/recipes-mender/mender-artifact/mender-artifact_3.8.0.bb (hunk 1 above)
Comment (lluiscampos) [reply meta-mender-core/recipes-mender/mender-artifact/mender-artifact_3.8.0.bb:103]: Yes, I figured. I also saw references to this script if/when the license checks fail. Nicely done.

**Comment by kacf** [comment]
> I tried but I couldn't spot anything wrong. Great work overall.

But the attempt is, as always, appreciated! 😃

**Pull request merged**